project/SmallJDK.scala

```diff
@@ -27,12 +28,17 @@ object SmallJDK {
     "java.naming",
     "java.net.http",
     "java.rmi",
+    "jdk.attach",
     "jdk.charsets",
     "jdk.crypto.ec",
     "jdk.httpserver",
     "jdk.localedata"
   )
 
+  private val ADDITIONAL_NI_BUILD_MODS = Seq(
+    "java.prefs"
```

Comment:
- I'd like us to fix GraalVM, so we don't need to add this `java.prefs` module workaround with GraalVM 25.0.1

Reply:
It should be supported in Bazel, and on all three platforms.